pkg/minikube/node/start.go

```diff
@@ -731,3 +737,17 @@ func addCoreDNSEntry(runner command.Runner, name, ip string, cc config.ClusterCo
 
 	return nil
 }
+
+// prints a warning to the console against the use of the 'virtualbox' driver, if alternatives are available
+func warnVirtualBox() {
+	var altDriverList strings.Builder
+	for _, choice := range driver.Choices(true) {
+		if choice.Name != "virtualbox" && choice.Priority != registry.Discouraged && !(choice.Name == "vmware" && !(choice.State.Installed && choice.State.Healthy)) {
+			altDriverList.WriteString(fmt.Sprintf("\n\t- %s", choice.Name))
+		}
+	}
+
+	if altDriverList.Len() != 0 {
+		out.Boxed("There are alternative drivers to virtualbox for better performance and support, consider using them {{.drivers}} \nTo turn this warning off use `minikube config set WantVirtualBoxDriverWarning false`", out.V{"drivers": altDriverList.String()})
```

Comment:
how about we improve the wording to be shorter and easier to get their attention
```
You have selected Virtualbox driver,  but there are better options, 
for better performance and support consider using a different driver:

To turn off this suggestion run
`minikube config set WantVirtualBoxDriverWarning false`

```